docs-ref-conceptual/index.yml

```diff
@@ -37,7 +37,7 @@ sections:
         src: https://docs.microsoft.com/java/docs-ref-conceptual/media/spring.svg
       title: Spring on Azure
     - href: java-azure-tools.md
-      html: <p>IDE plugins &amp; Tools</p>
+      html: <p>Supported JDKs &amp; IDE plugins</p>
```

Comment:
Wdyt of "Tools, IDEs, and JDKs"

Reply:
Great change. Made it into the current version